### ISSUE TITLE
Row/Column spams warnings in react 15.2.0

### ISSUE THREAD

**Issue by cythrawll** [nordsoftware/react-foundation#15]
### What did you do?\

use Column and Row properties with properties such as isColumn, offsetOnMedium, etc...

### What did you expect to happen?

react 15.2.0 not to throw warnings

### What actually happened?

spams Unknown Prop Warnings

### What version of this module are you using?

^0.6.8

I imagine this might be an issue throughout the library, but Row and Column were the only components in use right now.

https://github.com/facebook/react/pull/6800

**Comment by supasympa** [comment]
+1

**Comment by supasympa** [comment]
I had a look and it looks like the props are being transferred to the rendered element. In the case of Column I found that changing the following fixed the issue, but I'm not sure if it is the intended behaviour:

in **src/components/grid.js**

**BEFORE:**
```
   ...
    return (
          <div {...removeProps(props, ['offset'])} className={className}/>
        ); 
    ...
```

**AFTER:**
```
   ...
    return (
          <div {...removeProps(props, ['offset', 'small', 'medium', 'large'])} className={className}/>
        ); 
    ...
```

**Comment by supasympa** [comment]
in the case of Menu:

**ADD:**
**const { isSimple, alignment, ...rest } = props;**

```

export const Menu = props => {
  const { isSimple, alignment, ...rest } = props;
  const className = createClassName(
    props.noDefaultClassName ? null : 'menu',
    props.className,
    {
      'align-right': props.alignment === MenuAlignments.RIGHT,
      'align-center': props.alignment === MenuAlignments.CENTER,
      'icon-top': props.iconsOnTop,
      'expanded': props.isExpanded,
      'vertical': props.isVertical,
      'simple': props.isSimple,
      'nested': props.isNested,
      'dropdown': props.isDropdown,
      'medium-horizontal': props.horizontalOnMedium
    },
    generalClassNames(props)
  );

  return (
    <ul {...rest} className={className}/>
  );
};
```

**Comment by nitelite** [comment]
+1. Would love to see this issue resolved.

**Comment by bithavoc** [comment]
Is anyone working on this?

**Comment by crisu83** [comment]
We are also really annoyed by this as we use this library for numerous projects. The only problem is that while I know exactly what is causing the issue, it's a little bit hard to get rid of, mainly because it requires quite major refactoring at the core of this library. I will try to allocate time to this issue as soon as possible. Sorry for the inconvenience.

**Comment by crisu83** [comment]
This has now been fixed in version `0.7.0`.
